user_tools/src/spark_rapids_pytools/common/utilities.py

```diff
@@ -229,6 +230,23 @@ def get_and_setup_logger(cls, type_label: str, debug_mode: bool = False):
             logger.addHandler(fh)
         return logger
 
+    @classmethod
+    def modify_log4j_properties(cls, file_path: str, new_log_file_path: str) -> str:
+        timestamp = datetime.datetime.now().strftime('%Y%m%d%H%M%S')
+        new_log_file_with_timestamp = f'{new_log_file_path}/core_tools_log_{timestamp}.log'
```

Comment:
logfile will be saved in <outputdirectory>/apids_4_spark_qualification_output/ directory.
Open for suggestion for log file. Currently it will be something like `core_tools_log_20240808153410.log`

Reply:
In order to grab the log of the latest run, the user should pick the file name with largest timestamp?
if we assume that the most common case is CLI. then the if the log_file is saved in the parent directory of rapids_4_spark_qualification_output. then we won't need to give it a timestamp because the timestamp is implicit by the parent directory.